python/tvm/rpc/tracker.py

```diff
@@ -134,7 +134,8 @@ def put(self, value):
         self._schedule()
 
     def request(self, user, priority, callback):
-        heapq.heappush(self._requests, (-priority, time.time(), callback))
+        heapq.heappush(self._requests, (-priority, self._request_cnt, callback))
+        self._request_cnt += 1
```

Comment:
As we will use `threading.Thread` to request tracker, then we will call request. I think we should use lock

Reply:
Added a lock to make heappush atomic. I didn't make heappop atomic since this PR doesn't change its behavior. Let me know if there will be an issue.